form-validation.js

```diff
@@ -0,0 +1,14 @@
+const form = document.getElementById('form');
+const userEmail = document.getElementById('email');
+const small = userEmail.parentNode.querySelector('small');
+
+form.addEventListener('submit', (e) => {
+  if (userEmail.value !== userEmail.value.toLowerCase()) {
+    e.preventDefault();
```

Comment:
- [ ] You should validate the email address so that it does not have Upper case letters. Currently, however, the form submitted even though I used Upper case letter in the email.
![image](https://user-images.githubusercontent.com/44057863/169666505-d7fc02bf-45ef-4f20-9ea9-787530766dcb.png)

- [ Optional ] I think this line should be in first line of this event listener to prevent default behavior of form submission which is to post to URL in the action property.